HW7/Mengyuan_HW7.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
module name should all be in lowercase.
Quote from PEP 8:
"Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged."